language/stdlib/modules/LibraAccount.move

```diff
@@ -1422,6 +1491,17 @@ module LibraAccount {
         aborts_if !LibraTransactionPublishingOption::spec_is_script_allowed(sender, script_hash) with Errors::INVALID_STATE;
     }
 
+    spec define prologue_guarantees(sender: signer) : bool {
+        let addr = Signer::spec_address_of(sender);
+        LibraTimestamp::is_operating() && exists_at(addr) && !AccountFreezing::account_is_frozen(addr)
+    }
+
+    /// Used in transaction script to specify pre-conditions checked by the prologue.
+    spec schema TransactionCertified {
```

Comment:
Is "TransactionCertified" a good name?  I don't understand it.  PrologueGuarantees or something like that would be more immediately understandable.

Reply:
What about `IsValidTransaction`? Emma and I discussed the "guarantees" yesterday, and it is also not to good. The `IsValid` seems to be the most neutral.